### ISSUE TITLE
SVG support

### ISSUE THREAD

**Issue by pmros** [joaomilho/act#12]
It seems Act doesn't support SVG yet.

This simple code doesn't work:
```javascript
main(['svg', {height: 100, width: 100}, [
  ['circle', {cx: 50, cy: 50, r: 40}]
]])
```
Attributes are not set.

I'd like Act would support SVG (virtual-dom supports it). 
Thank you for Act, it's very nice.

**Comment by joaomilho** [comment]
Yes, I still haven't implemented SVG, but it was part of the "todo" document, I just didn't have the time to move it to an issue, so now we have one :D.

Given Act is based on virtual-dom (at least for now, check #4 ), and virtual-dom has support for SVG it can be done. The difficulty is that it requires to set some namespaces of tags, and I would like to make it completely transparent. One thing I was looking into is https://github.com/substack/virtual-hyperscript-svg, but still couldn't devote much time.

If you feel you have some time and wanna contribute, I would love to give you some pointers.

**Comment by pmros** [comment]
I've been playing a while with Act and SVG. In adition to set SVG namespace (http://www.w3.org/2000/svg), you need to set attributes (not properties). So this simple code does work:
```javascript
main(['svg', {namespace: "http://www.w3.org/2000/svg", attributes: {height: 100, width: 100}},
  [
    ['circle', {namespace: "http://www.w3.org/2000/svg", attributes: {cx: 50, cy: 50, r: 40}}]
  ]
])
```
I'll try to patch [jsonToVirtualDOM.js](https://github.com/joaomilho/act/blob/master/packages/main/internals/jsonToVirtualDOM.js).

**Comment by joaomilho** [comment]
I suggest instead of patching it on main, to creating a new package, say `@act/svg`, and svg's main will delegate the render to a modified jToVDOM that will append namespace/attributes to a set of tags (svg, circle, path...).

One thing we still have to solve is how do you compose @act packages. Say you wanna use svg+animations+hmr. Today that's not yet doable, but I suppose it will be easy to solve.

**Comment by pmros** [comment]
I don't know if `@act/svg` is the best solution since you can embed multiple svg elements inside html document. Furthermore, you are repeating yourself: svg tag is enough, you should not take care about importtng a special svg package.

**Comment by joaomilho** [comment]
Fixed by #15

**Comment by joaomilho** [comment]
I agree, specially after seeing how simple was the actual implementation.
